samples/net/rpl_border_router/README.rst

```diff
@@ -0,0 +1,169 @@
+.. _rpl-border-router-sample:
+
+RPL Border Router
+#################
+
+Overview
+********
+
+The RPL border router sample application for Zephyr provides a HTTP(S) server
+and net shell for management purposes. Typically it would be used to connect to
+IEEE 802.15.4 network but Bluetooth IPSP network functionality is also possible.
+
+The source code for this sample application can be found at:
+:file:`samples/net/rpl_border_router`.
+
+Requirements
+************
+
+- Real device like Freedom Board (FRDM-K64F) with MCR20A IEEE 802.15.4 support.
+- Linux machine with web browser and the screen terminal emulator (optional).
+- Ethernet access for management purposes (optional).
+
+Note that there is no support for running RPL border router in QEMU, as the
+border router requires access to a real radio network technology like
```

Comment:
We need more than one node in the network. The qemu emulation for 15.4 is one node only solution. This could be done in simulated way with emul8 but that is some future work.